docs/en/test_cards/mc_07_optical_flow_failure.md

```diff
@@ -0,0 +1,44 @@
+# Test MC_07 - Optical Flow Failure
+
+## Objective
+
+To test that optical flow works as expected with a low mounted optical flow sensor
+
+## Preflight
+
+Ensure that the drone's optical flow sensor is mounted less than an inch off of the ground.
```

Comment:
```suggestion
Ensure that the drone's optical flow sensor is mounted more than an inch off of the ground.
```
this shouldn't be a strict requirement, we should do what we do for rangefinder and "fake it" if the signal quality is 0 while on the ground. This will allow takeoff with the assumption that flow will become valid as soon as you've got enough ground clearance